src/components/QrScanButton.tsx

```diff
@@ -1,45 +1,23 @@
 import React from 'react'
-import { StyleProp, StyleSheet, View, ViewStyle } from 'react-native'
+import { StyleProp, ViewStyle } from 'react-native'
 import { QrScreenEvents } from 'src/analytics/Events'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
-import Touchable from 'src/components/Touchable'
 import ScanIcon from 'src/icons/ScanIcon'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
-import { Spacing } from 'src/styles/styles'
+import { TopBarIconButtonV2 } from 'src/navigator/TopBarIconButtonV2'
 
 interface Props {
   style?: StyleProp<ViewStyle>
   size?: number
   testID?: string
 }
 
-export default function QrScanButton({ style, size, testID }: Props) {
```

Comment:
Is there a reason `style` and `size` were removed, or should these be included (size for icon, style for TopBarIconButtonV2) as above?